embassy-rp/src/uart/mod.rs

```diff
@@ -1264,6 +1264,26 @@ impl<'d, T: Instance, M: Mode> embedded_hal_nb::serial::Write for Uart<'d, T, M>
     }
 }
 
+impl<'d, T: Instance> embedded_io::ErrorType for Uart<'d, T, Blocking> {
+    type Error = Error;
+}
+
+impl<'d, T: Instance> embedded_io::Read for Uart<'d, T, Blocking> {
```

Comment:
the raw uart can't impl Read, see here for reasons: https://github.com/embassy-rs/embassy/pull/2185#issuecomment-1810047043

can you remove it? Write looks good.

Reply:
I've removed it, thank you!